pkg/controller/handler.go

```diff
@@ -170,11 +171,12 @@ func (h *handler) CreateServiceBinding(in *servicecatalog.Binding) (*servicecata
 	// Assign UUID to binding.
 	in.Spec.OSBGUID = uuid.NewV4().String()
 
+	// TODO: uncomment parameters line once parameters types are refactored.
```

Comment:
Can you clarify this? Seems like it is breaking the system as it exists today. Or is it already non-functioning?